### PR TITLE
Feature/css responsiveness

### PR DESCRIPTION
**Pull request by sydnerd**
# Pull Request Template
## Description

Added some media queries to make pages more responsive

## Type of change
- [ ] Bug fix 
- [ ] New feature
- [X] Refactoring

## Checklist:
- [X] My code follows the Turing's guidelines of this project
- [X] My changes generate no new errors/warnings

### CODE REVIEW

**Comment by sydnerd** [comment]
I like that you added a message in the my bird section when there aren't any birds! Great touch.

File: src/components/App/App.js [hunk 1]
```diff
@@ -40,6 +40,8 @@ const App = () => {
     <h2>Loading your birds...</h2>
   );
 
+  const noMyBirdMsg = !myBirds.length && <h2>GO FIND SOME BIRDS KIDDO!</h2>
```
Comment (LKessell) [inline src/components/App/App.js:43]: Love this!

File: src/components/App/App.css [hunk 2]
```diff
@@ -43,9 +30,29 @@ path:hover {
...
-  height: 10vh;
+  height: 8vh;
 }
 
 .hidden {
   display: none;
 }
+
+/* Small Tablet Styling */
+@media screen and (max-width: 620px) and (orientation: portrait) {
```
Comment (LKessell) [inline src/components/App/App.css:41]: This looks so good on small screens!

File: src/components/StatePicker/StatePicker.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.us-state-map {
```
Comment (LKessell) [inline src/components/StatePicker/StatePicker.css:1]: These classes make more sense here, good job!

**Review by LKessell — APPROVED** [review]
Looks awesome, the responsiveness makes it look super polished. ☺️

**Pull request merged**